api/types/image.go

```diff
@@ -12,10 +12,15 @@ type Image struct {
 	// NewName is the value used to replace the original name.
 	NewName string `json:"newName,omitempty" yaml:"newName,omitempty"`
 
+	// TagSuffix is the value used to suffix the original tag
+	// If digest or newtag is present TagSuffix value is ignored
+	TagSuffix string `json:"tagSuffix,omitempty" yaml:"tagSuffix,omitempty"`
+
 	// NewTag is the value used to replace the original tag.
+	// If newtag is present TagSuffix value is ignored
 	NewTag string `json:"newTag,omitempty" yaml:"newTag,omitempty"`
 
 	// Digest is the value used to replace the original image tag.
```

Comment:
Can we update these comments to explain the difference between `digest` and `newTag`? As it is, it sounds like they should do the exact same thing but that is not the case.